egograph/backend/infrastructure/database/chat_connection.py

```
@@ -5,12 +5,13 @@
 
 import logging
 import sqlite3
-from pathlib import Path
+
+from egograph_paths import CHAT_SQLITE_PATH
 
 logger = logging.getLogger(__name__)
 
-# チャット履歴DBのパス（backend/data/chat.sqlite）
-DB_PATH = Path(__file__).parent.parent.parent / "data" / "chat.sqlite"
+# チャット履歴DBのパス（repo の兄弟 data/backend/chat.sqlite）
```

Ruff (0.15.9)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)